core/runtime/src/test/java/io/quarkus/runtime/configuration/DotEnvTestCase.java

```diff
@@ -29,21 +29,17 @@
 public class DotEnvTestCase {
 
     static ClassLoader classLoader;
-    static ConfigProviderResolver cpr;
+    static ConfigProviderResolver configProviderResolver;
```

Comment:
I think we can drop all of the Config registration and just use SmallRyeConfig locally at each test method.